dScripts/02_server/Map/NS/Waves/ZoneNsWaves.cpp

```diff
@@ -437,7 +437,7 @@ std::vector<Wave> ZoneNsWaves::GetWaves() {
 																									}, {}, {},
 																									"Treasure_Camera",
 																									5.0f,
-																									(uint32_t)-1,
+																									static_cast<uint32_t>(-1),
```

Comment:
yeah @aronwk-aaron i can see how this looks silly lol

Reply:
Any change I should make?

Reply:
Nah, just an observation as to why casting unsigned to signed looks weird